tests/citeproc.rs

```diff
@@ -638,3 +638,50 @@ fn access_date() {
         .unwrap();
     assert_eq!(buf, "Retrieved 2021, from https://example.com/");
 }
+
+#[test]
```

Comment:
Ok

Reply:
I have a problem after rebase, which tells me I can't build biblatex because of duplicated definition in its internals. As far as I understand, problem is not on my side as an error occurs on library compilation. Here's the logs:
```rust
error[E0592]: duplicate definitions with name `language`yaml, biblatex, citationberg
   --> /Users/find.art.in.living/.cargo/registry/src/index.crates.io-6f17d22bba15001f/biblatex-0.9.1/src/macros.rs:5:13
    |
5   |               pub fn $name(&self) -> Result<fields!(@ret $($ret)?), RetrievalError> {
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |               |
    |               duplicate definitions for `language`
    |               other definition for `language`
    |
   ::: /Users/find.art.in.living/.cargo/registry/src/index.crates.io-6f17d22bba15001f/biblatex-0.9.1/src/lib.rs:811:5
    |
811 | /     fields! {
812 | |         abstract_: "abstract",
813 | |         addendum: "addendum",
814 | |         afterword: "afterword" => Vec<Person>,
...   |
875 | |         gender: "gender" => Gender,
876 | |     }
    | |_____- in this macro invocation
    |
    = note: this error originates in the macro `fields` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0592]: duplicate definitions with name `set_language`
   --> /Users/find.art.in.living/.cargo/registry/src/index.crates.io-6f17d22bba15001f/biblatex-0.9.1/src/macros.rs:30:13
    |
22  |               pub fn [<set_ $name>](&mut self, item: Chunks) {
    |               ---------------------------------------------- other definition for `set_language`
...
30  |               pub fn [<set_ $name>](&mut self, item: $ty) {
    |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ duplicate definitions for `set_language`
    |
   ::: /Users/find.art.in.living/.cargo/registry/src/index.crates.io-6f17d22bba15001f/biblatex-0.9.1/src/lib.rs:811:5
    |
811 | /     fields! {
812 | |         abstract_: "abstract",
813 | |         addendum: "addendum",
814 | |         afterword: "afterword" => Vec<Person>,
...   |
875 | |         gender: "gender" => Gender,
876 | |     }
    | |_____- in this macro invocation
    |
    = note: this error originates in the macro `fields` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0592`.
error: could not compile `biblatex` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

Reply:
Works on latest version.